### PR TITLE
Update BrightID copy

### PR DESCRIPTION
**Pull request by samajammin**
https://www.notion.so/efdn/Review-BrightID-flow-content-362143fd91bc47ac9f0959afc9a5cf7e

### CODE REVIEW

**Comment by samajammin** [comment]
@corwintines tagging you for review 😄

File: vue-app/src/views/Verify.vue [hunk 1]
```diff
@@ -183,11 +188,11 @@
             />
           </div>
           <div v-if="currentStep === 3">
-            <h2 class="step-title">Get registered</h2>
+            <h2 class="step-title">Register</h2>
             <p>
               To protect the round from bribery and fraud, you need to add your
-              wallet address to a smart contract register. Once you’re done, you
-              can join the funding round!
+              wallet address to a smart contract registery. Once you’re done,
```
Comment (samajammin) [inline vue-app/src/views/Verify.vue]:
```suggestion
              wallet address to a smart contract registry. Once you’re done,
```

File: vue-app/src/views/Verify.vue [hunk 2]
```diff
@@ -70,19 +70,26 @@
...
-                @click="handleIsVerifiedClick"
-                class="btn-primary"
-                :disabled="loadingManualVerify"
-              >
-                Check if you are verified
-              </button>
-              <loader v-if="loadingManualVerify" />
+            <div v-if="!isManuallyVerified">
+              <p>
+                Once you're verified in BrightID, verification confirmation
```
Comment (corwintines) [inline vue-app/src/views/Verify.vue:78]:
```suggestion
                Verification confirmation
```
This seems a little contradictory saying once your verified check verification. Maybe just seems weird to me so feel free to push back, but could probably just get rid of the first part of this sentence.

Comment (corwintines) [reply vue-app/src/views/Verify.vue:78]: On second thought, I think this makes sense.

**Review by corwintines — APPROVED** [review]

**Pull request merged**